src/tests-2.5/global-setup.ts

```diff
@@ -40,6 +40,7 @@ export default async (): Promise<void> => {
   process.env.PG_DATABASE = 'postgres';
   process.env.STACKS_CHAIN_ID = '0x80000000';
 
+  await migrate('down');
```

Comment:
This needed for running multiple tests with the same postgres?

Reply:
The `/test-2.5/*` tests are one shot, only one can run since they pollute the stacks-node and bitcoind. Preserving postgres state makes no difference